examples/vision/md/supervised-contrastive-learning.md

```diff
@@ -9,13 +9,11 @@
 <img class="k-inline-icon" src="https://colab.research.google.com/img/colab_favicon.ico"/> [**View in Colab**](https://colab.research.google.com/github/keras-team/keras-io/blob/master/examples/vision/ipynb/supervised-contrastive-learning.ipynb)  <span class="k-dot">•</span><img class="k-inline-icon" src="https://github.com/favicon.ico"/> [**GitHub source**](https://github.com/keras-team/keras-io/blob/master/examples/vision/supervised-contrastive-learning.py)
 
 
-
----
 ## Introduction
 
 [Supervised Contrastive Learning](https://arxiv.org/abs/2004.11362)
 (Prannay Khosla et al.) is a training methodology that outperforms
-plain crossentropy-supervised training on classification tasks.
+supervised training on classification tasks with cross-entropy.
```

Comment:
In the Keras API, "crossentropy" is a single word

Reply:
Got it @fchollet